metron-analytics/metron-profiler/README.md

```diff
@@ -275,27 +280,105 @@ The Profiler runs as an independent Storm topology.  The configuration for the P
+| [`profiler.hbase.table`](#profilerhbasetable)                                 | The name of the HBase table that profiles are written to.
+| [`profiler.hbase.column.family`](#profilerhbasecolumnfamily)                  | The column family used to store profiles.
+| [`profiler.hbase.batch`](#profilerhbasebatch)                                 | The number of puts that are written to HBase in a single batch.
+| [`profiler.hbase.flush.interval.seconds`](#profilerhbaseflushintervalseconds) | The maximum number of seconds between batch writes to HBase.
 
-```
-$ $METRON_HOME/start_profiler_topology.sh
-```
+### `profiler.input.topic`
+
+*Default*: indexing
+
+The name of the Kafka topic from which to consume data.  By default, the Profiler consumes data from the `indexing` topic so that it has access to fully enriched telemetry.
+
+### `profiler.output.topic`
+
+*Default*: enrichments
+
+The name of the Kafka topic to which profile data is written.  This property is only applicable to profiles that define  the [`triage` result field](#result).  This allows Profile data to be selectively triaged like any other source of telemetry in Metron.
+
+### `profiler.period.duration` 
+
+*Default*: 15
+
+The duration of each profile period.  This value should be defined along with [`profiler.period.duration.units`](#profilerperioddurationunits). 
+
+*Important*: To read a profile using the [Profiler Client](metron-analytics/metron-profiler-client), the Profiler Client's `profiler.client.period.duration` property must match this value.  Otherwise, the Profiler Client will be unable to read the profile data.  
+
+### `profiler.period.duration.units` 
+
+*Default*: MINUTES
+
+The units used to specify the `profiler.period.duration`.  This value should be defined along with [`profiler.period.duration`](#profilerperiodduration). 
+
+*Important*: To read a profile using the Profiler Client, the Profiler Client's `profiler.client.period.duration.units` property must match this value.  Otherwise, the [Profiler Client](metron-analytics/metron-profiler-client) will be unable to read the profile data. 
+
+### `profiler.workers`
+
+*Default*: 1
+
+The number of worker processes to create for the Profiler topology.  This property is useful for performance tuning the Profiler.
+
+### `profiler.executors`
+
+*Default*: 0
+
+The number of executors to spawn per component for the Profiler topology.  This property is useful for performance tuning the Profiler. 
+
+### `profiler.ttl`
+
+*Default*: 30
+
+ If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with [`profiler.ttl.units`](#profilerttlunits).
+ 
+ This property does not affect the persisted Profile data in HBase, only the state stored in memory during the execution of each profile period.  
```

Comment:
Suggest: This timeout does not affect the persisted Profile data in HBase, but any state stored in memory during the execution of the last profile period will be deleted.